study.md

```diff
@@ -74,13 +79,13 @@ What are the similarities and differences between Ruby's and JavaScript's
 Similarities:
 
 ```md
-<!-- your answer here -->
+For both Ruby and JS, the unshift method adds one or more elements to the beginning of an array.
 ```
 
 Differences:
 
 ```md
-<!-- your answer here -->
+I was unable to determine any significant differences for the unshift method with JS and Ruby (aside from syntax).
```

Comment:
Ruby's unshift array method returns the modified array, allowing method
chaining.

JavaScript's unshift array method returns the length of the modified array.